setup_once.h

```diff
@@ -397,9 +397,9 @@ typedef int sig_atomic_t;
  */
 
 #ifdef DEBUGBUILD
-#define DEBUGF(x) x
+#define DEBUGF(channel, ...) ares_debug(channel, __VA_ARGS__)
```

Comment:
@bagder, does c-ares still support pre-C99 platforms?

IIRC variadic macros are a >=C99 feature, so if we still need to target compilers that old then we'll need a different approach -- maybe `DEBUGF(ares_debug(channel, msg))` or maybe have plain `ares_debug(channel, msg)` and move the `#ifdef DEBUGBUILD` into there.